docs/guide/mobile-best-practices.md

```diff
@@ -0,0 +1,40 @@
+# Mobile Best Practices
+
+If your site works with the MetaMask Extension, then you should be all set!
+If it doesn't, please refer to our [best practices](./mobile-best-practices.html) and if you're still having issues, please open an issue [here](https://github.com/MetaMask/metamask-mobile)
+
+# ethereum.initialized
```

Comment:
Is this section the provider section by @rekmarks ?

Note: best practices link is broken.

Reply:
Dapp compatibility to include MetaMask mobile as a listed wallet, and that link to open MetaMask app (if installed) or to go to play/app store (if not yet installed)

Simple guided-steps on letting your user use your apps on Metamask

Message to web3 app builders:
- send your users to Metamask app listing
- add Metamask mobile as option to connect wallet
- Support MM inpage provider and test so it works well